src/runtime/audience-action-flow-test.js

```diff
@@ -132,6 +135,10 @@ describes.realWin('AudienceActionFlow', {}, (env) => {
       .expects('set')
       .withExactArgs(Constants.USER_TOKEN, 'fake user token', true)
       .exactly(1);
+    storageMock
+      .expects('set')
+      .withExactArgs(Constants.READ_TIME, nowAsSecondsString(), false)
```

Comment:
Instead of calling the function directly, lets compare against a constant string value

Reply:
Done.